src/main/webapp/trigger/help-GerritTriggerConfiguration.html

```diff
@@ -8,7 +8,9 @@
     You can specify the name pattern in three different ways, as provided by the &quot;Type&quot; drop-down menu.
 </p>
 <ul>
-    <li><strong>Plain:</strong> The exact name in Gerrit, case sensitive equality.</li>
+    <li><strong>Plain:</strong> The exact name in Gerrit, case insensitive equality.</li>
+    <li><strong>PlainVar:</strong> The exact name in Gerrit, case insensitive equality, allows usage of system environment
```

Comment:
I doubt it is on the gerrit host? You me4an the Jenkins master right? Hard to say since this PR seems incomplete.